src/sagemaker/image_uri_config/autogluon.json

```diff
@@ -345,6 +346,41 @@
                 "repository": "autogluon-training",
                 "processors": ["cpu", "gpu"],
                 "py_versions": ["py39"]
+            },
+            "0.8.2": {
```

Comment:
Please update issue description , Merge Checklist  etc 

Changes are LGTM at high level. PR tests and Notebook test failure unrelated to this change, tracking internally

Reply:
Updated